SU-HSDO/suhumsci
# Arch 89  ARCH-86 datefield checkbox & Viewfield select

# READY FOR REVIEW

# Summary
- Added "Show End Date" checkbox to date range widget
- disable materialize for viewfield display id select list.

# Needed By (Date)
- End of sprint

# Urgency
- Low

# Steps to Test

1. Checkout this branch
2. sync you local environment
3. go to create an event
4. verify the date range field now only shows the start date and a checkbox.
5. test out and make sure that the date range still works correctly
6. create a basic page and add a view field paragraph item
7. choose a view
8. verify the "Display" displays a select list correctly.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
